pretrain.py

```diff
@@ -74,11 +76,17 @@ def gen_func(images):
 model = network.create(input_shape, args.network)
 network.compile(model, args.optimizer)
 
-model.save('pretrain.h5')
+# serialize model to JSON
+model_json = model.to_json()
+with open('model_pretrain.json', 'w') as json_file:
+    json_file.write(model_json)
 
-model.fit_generator(train_iterator, validation_data=validation_iterator, steps_per_epoch=train_couples_len / BATCH_SIZE, validation_steps=validation_couples_len / BATCH_SIZE, epochs=EPOCHS)
+filepath = 'best_pretrain_weights.hdf5'
```

Comment:
Remove this variable and use the value directly in the `ModelCheckpoint` constructor.